packages/framework/src/index.ts

```diff
@@ -1,7 +1,3 @@
 export { Client } from './client';
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** We need to bring back the `CronExpression` export as it's a publicly documented utility that folks are already using.‏
```suggestion
export { Client } from './client';
export { CronExpression } from './constants';
```

Reply:
Thanks for the reminder. Fixed.